tests/providers/docker/decorators/test_docker.py

```diff
@@ -15,16 +15,41 @@
 # specific language governing permissions and limitations
 # under the License.
 
+import sys
+import tempfile
+from base64 import b64decode
+from unittest.mock import patch
+
 from airflow.decorators import task
 from airflow.models.dag import DAG
+from airflow.providers.docker.operators.docker import DockerOperator
 from airflow.utils import timezone
 
 DEFAULT_DATE = timezone.datetime(2021, 9, 1)
 
+DOCKER_IMAGE = "quay.io/bitnami/python:3.9"
+
+
+def docker_execute(self, context):
+    code = b64decode(self.environment['__PYTHON_SCRIPT'])
+    args = b64decode(self.environment['__PYTHON_INPUT'])
+    with tempfile.NamedTemporaryFile() as f:
+        with tempfile.NamedTemporaryFile() as g:
```

Comment:
```suggestion
    with tempfile.NamedTemporaryFile() as f, tempfile.NamedTemporaryFile() as g:
```

Reply:
Well, not to be pedantic but the inner context is only used during execution while the outer one is used afterwards to learn about the result ...

Reply:
Fair enough.